sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-spring-security-resource-server/README.md

```diff
@@ -0,0 +1,113 @@
+# OAuth 2.0 Sample for Azure AD Spring Boot Starter Resource Server library for Java
+
+## Key concepts
+This sample illustrates how to protect a Java web API by restricting access to its resources to authorized accounts only.
+
+1. The bearer token is obtained from the request header. 
+2. `JwtDecoder`  is used to parse the token into `Jwt`.
+3. Claims, Headers etc in `Jwt` will be extracted, they will be wrapped in `AzureOAuth2AuthenticatedPrincipal` object.
+4. `AzureOAuth2AuthenticatedPrincipal` will eventually be set into SecurityContext.
+
+
+## Getting started
```

Comment:
We refactor this and [instruction] below in @moarychan's PR, please follow this https://github.com/Azure/azure-sdk-for-java/pull/17862 to change.